src/lib/orion_websocket/ws.cpp

```diff
@@ -97,7 +102,7 @@ static int wscllbck(libwebsocket_context *ctx,
       unsigned char *p = &buff[LWS_SEND_BUFFER_PRE_PADDING];
 
       sprintf((char*)p, "%s", dat->message);
-      libwebsocket_write(ws, p, strlen(dat->message), LWS_WRITE_TEXT);
+      lws_write(ws, p, strlen(dat->message), LWS_WRITE_TEXT);
```

Comment:
This I like, lots of bytes 'saved' here.
I would call it wsWrite though.
Matter of opinion.

NTC